adoc/extensions/sycl_khr_command_graph.adoc

```diff
@@ -1,9 +1,24 @@
 = sycl_khr_command_graph
 
+== Overview
+
+The command graph abstraction defines an explicitly lazy execution model by
+decoupling command creation from submission. These concepts are potentially
+tied together in SYCL implementations that eagerly submit work to a device when
+a command-group is submitted to a queue. By providing a prior construction
+stage before a workload of commands begins execution on the device, submission
+of multiple commands can happen in a single submission call instead of many,
+reducing the overhead of multiple independent command submissions. Deferral of
+command submission can also allow for the SYCL implementation to optimize
+dependencies within the user defined graph, which may improve concurrency
+and other performance metrics.
+
 == Dependencies
 
 This extension has no dependencies on other extensions.
 
+== Contributors
+
 == Feature test macro
```

Comment:
nitpicking, but a chance to make capatalization consistent across section headings
```suggestion
== Feature Test Macro
```